packages/cf-component-tabs/example/basic/component.js

```diff
@@ -1,5 +1,15 @@
 import React from 'react';
-import { Tabs, TabsPanel } from 'cf-component-tabs';
+import {
+  Tabs,
+  TabsItemUnstyled,
+  TabsItemTheme,
+  TabsPanel
+} from 'cf-component-tabs';
+import { applyTheme } from 'cf-style-container';
+
+const Tab = applyTheme(TabsItemUnstyled, TabsItemTheme, () => ({
+  background: '#FFFFF0'
```

Comment:
Where is this yellowy-white color being used exactly? Seems like it could actually be white... but a screenshot/additional context would help. :)